docs/networks/advanced-guides/withdrawal-update.md

```diff
@@ -199,24 +206,48 @@ The **Withdrawal Credential** can be shared directly from your node or the publi
   </TabItem>
   <TabItem value="broadcast-from-node" label="Broadcast Message from Node">
 
-:::caution
-
-Within the following section, `<PATH_TO_JSON_FILE>` has to be exchanged with the path of the `bls_to_execution_change.json` file.
+1. Print and copy the contents of the `bls_to_execution_change.json` file:
 
-:::
+```bash
+cat bls_to_execution_change.json
+```
 
-<div>
+2. Input the contents into the broadcast command:
 
-1. Log into your node’s terminal
-2. Copy the `bls_to_execution_change.json` file to your node
-3. Send the JSON file to the local consensus instance using the following code:
+```bash
+POST -H "Content-type: application/json" -d  '<MY_CONTENTS>'
+http://localhost:3500/eth/v1/beacon/pool/bls_to_execution_changes
```

Comment:
I'm on mobile so I don't see the diff well. But are you explaining where this localhost is running? I see you have removed some steps that tell users to log into the node etc.

Reply:
explained right below: "3. Log into your node’s terminal and execute the broadcast command on its local consensus interface:"

Good catch- the HTTP is not part of the actual message, but rather the destination. I removed it from step 2 and only added it in step 3 👍🏻

Also added more context to the delay. Ready to be merged now.